pkgs/data/fonts/nerdfonts/default.nix

```diff
@@ -25,33 +26,97 @@ let
     fName:
     fontsShas."${fName}"
   );
-  srcs = lib.attrsets.mapAttrsToList (
+  # Map raw font names into clean attribute names
+  attrNames = {
+    "3270" = "_3270";
+    "Agave" = "agave";
+    "AnonymousPro" = "anonymous-pro";
+    "Arimo" = "arimo";
+    "AurulentSansMono" = "aurulent-sans-mono";
+    "BigBlueTerminal" = "big-blue-terminal";
+    "BitstreamVeraSansMono" = "bitstream-vera-sans-mono";
+    "CascadiaCode" = "cascadia-code";
+    "CodeNewRoman" = "code-new-roman";
+    "Cousine" = "cousine";
+    "DaddyTimeMono" = "daddy-time-mono";
+    "DejaVuSansMono" = "dejavu-sans-mono";
+    "DroidSansMono" = "droid-sans-mono";
+    "FantasqueSansMono" = "fantasque-sans-mono";
+    "FiraCode" = "fira-code";
+    "FiraMono" = "fira-mono";
+    "Go-Mono" = "go-mono";
+    "Gohu" = "gohu";
+    "Hack" = "hack";
+    "Hasklig" = "hasklig";
+    "HeavyData" = "heavy-data";
+    "Hermit" = "hermit";
+    "iA-Writer" = "ia-writer";
+    "IBMPlexMono" = "ibm-plex-mono";
+    "Inconsolata" = "inconsolata";
+    "InconsolataGo" = "inconsolata-go";
+    "InconsolataLGC" = "inconsolata-lgc";
+    "Iosevka" = "iosevka";
+    "JetBrainsMono" = "jetbrains-mono";
+    "Lekton" = "lekton";
+    "LiberationMono" = "liberation-mono";
+    "Meslo" = "meslo";
+    "Monofur" = "monofur";
+    "Monoid" = "monoid";
+    "Mononoki" = "mononoki";
+    "MPlus" = "mplus";
+    "Noto" = "noto";
+    "OpenDyslexic" = "open-dyslexic";
+    "Overpass" = "overpass";
+    "ProFont" = "profont";
+    "ProggyClean" = "proggy-clean";
+    "RobotoMono" = "roboto-mono";
+    "ShareTechMono" = "share-tech-mono";
+    "SourceCodePro" = "source-code-pro";
+    "SpaceMono" = "space-mono";
+    "Terminus" = "terminus";
+    "Tinos" = "tinos";
+    "Ubuntu" = "ubuntu";
+    "UbuntuMono" = "ubuntu-mono";
+    "VictorMono" = "victor-mono";
```

Comment:
I don't like this at all, sorry. Part of the point of https://github.com/NixOS/nixpkgs/pull/84858 was to also automate the process of fetching the available fonts there. This is a hand made dictionary I see no way of automating.

Reply:
Correction: ~~#84858~~ -> https://github.com/NixOS/nixpkgs/pull/84979

Reply:
Yes, I totally understand, and agree. This could also be automated by some rules in how to do the conversion but they'd be a bit comples and there would still be some exceptions (`DejaVu` is converted to `dejavu` instead of `deja-vu`).